docs/src/main/tut/typeclasses/lawtesting.md

```diff
@@ -74,72 +70,73 @@ but you won't need it if you import `org.scalacheck.ScalacheckShapeless._`).
+the `ScalaCheck` `Properties` provided by cats-laws.
 
-For other test frameworks, you need to resort to their integration with `ScalaCheck` to test
-the `ScalaCheck` `Properties` provided by cats-laws. 
-
-So here is the Scalatest test for our example, basically we import `cats.laws.discipline.FunctorTests` and 
-call the `checkAll` helper with it.
+The following example is for ScalaTest.  If not using 
 
 ```tut:book
-
-import cats.tests.CatsSuite	
+import cats.implicits._
 import cats.laws.discipline.FunctorTests
+import org.scalatest.funsuite.AnyFunSuite
+import org.typelevel.discipline.scalatest.Discipline
+import arbitraries._
 
-class TreeLawTests extends CatsSuite {
+class TreeLawTests extends AnyFunSuite with Discipline {
   checkAll("Tree.FunctorLaws", FunctorTests[Tree].functor[Int, Int, String])
 }
 ```
 
+* `cats.implicits._` imports the instances we need for `Eq[Tree[Int]]`, which the laws use to compare trees.
+* `FunctorTests` contains the functor laws.
+* `AnyFunSuite` defines the style of ScalaTest.
+* `Discipline` provides `checkAll`, and must be mixed into `AnyFunSuite`
+* `arbitraries._` imports the `Arbitrary[Tree[_]]` instances needed to check the laws.
+
 Now when we run `test` in our sbt console, ScalaCheck will test if the `Functor` laws hold for our `Tree` type.
 You should see something like this:
 
 ```
 [info] TreeLawTests:
-[info] - Tree.FunctorLaws.functor.covariant composition
-[info] - Tree.FunctorLaws.functor.covariant identity
-[info] - Tree.FunctorLaws.functor.invariant composition
-[info] - Tree.FunctorLaws.functor.invariant identity
+[info] - Tree.FunctorLaws.functor.covariant composition (58 milliseconds)
+[info] - Tree.FunctorLaws.functor.covariant identity (3 milliseconds)
+[info] - Tree.FunctorLaws.functor.invariant composition (19 milliseconds)
+[info] - Tree.FunctorLaws.functor.invariant identity (3 milliseconds)
 [info] ScalaTest
-[info] Run completed in 537 milliseconds.
+[info] Run completed in 1 second, 362 milliseconds.
 [info] Total number of tests run: 4
 [info] Suites: completed 1, aborted 0
 [info] Tests: succeeded 4, failed 0, canceled 0, ignored 0, pending 0
 [info] All tests passed.
 [info] Passed: Total 4, Failed 0, Errors 0, Passed 4
-[success] Total time: 1 s, completed Aug 31, 2017 2:19:22 PM
+[success] Total time: 2 s, completed Aug 2, 2019 12:01:17 AM
 ```
 
 And voila, you've successfully proven that your data type upholds the Functor laws!
 
-### Testing cats.kernel instances
+### Testing `cats.kernel` instances
 
 For most of the type classes included in Cats, the above will work great.
 However, the law tests for the type classes inside `cats.kernel` are located in `cats.kernel.laws.discipline.*` instead.
 So we have to import from there to test type classes like `Semigroup`, `Monoid`, `Group` or `Semilattice`.
 
-Let's test it out, by defining a `Semigroup` instance for our `Tree` type.
+Let's test it out by defining a `Semigroup` instance for our `Tree` type.
```

Comment:
Random grammar fixes in the middle of an otherwise pure refactoring will never cease to bemuse and satisfy me.